spritz/README.md

```diff
@@ -41,9 +41,6 @@ After getting your `LottieAnimationView` and your `ViewPager`, just create a `Sp
 ```java
 Spritz spritz = Spritz
                     .with(lottieAnimationView)
-                    .withDefaultSwipeAnimationDuration(300, TimeUnit.MILLISECONDS)
```

Comment:
No more needed, the animation will complete the swipe step according to the offset sent by the viewpager listener.